example/src/main/java/org/hibernate/example/reactive/MutinyMain.java

```diff
@@ -15,13 +15,22 @@
  */
 public class MutinyMain {
 
+	private static final String DEFAULT_PU = "postgresqlExamplePU";
+
+	// The first argument can be used to select a persistenceUnit.
+	// Check resources/META-INF/persistence.xml for available names.
+	public static String persitenceUnitName(String[] args) {
+		return args.length > 0 ? args[0] : DEFAULT_PU;
+	}
+
```

Comment:
I would prefer to move all this stuff down to the bottom of the file so it's not the first thing people see.

Also there's no need to make a `private static final String DEFAULT_PU` for a string literal that is used in exactly one place.